### ISSUE TITLE
Shuffled smart playlist start at album 1, track 1

### ISSUE THREAD

**Issue by staxas** [clementine-player/Clementine#6436]
### Before posting

Please follow the steps below and check the boxes with [x] once you did the step.

- [x] I checked the issue tracker for similar issues
- [x] I checked the [changelog](https://github.com/clementine-player/Clementine/blob/master/Changelog) if the issue is already resolved
- [x] I tried the latest Clementine build from [here](https://builds.clementine-player.org/)

### System information

Please provide information about your system and the version of Clementine used.

- Operating System:
Kubuntu 18.04
- Clementine version: 
1.3.1

### Expected behaviour / actual behaviour
Fully shuffled smart playlist / starts at album 1/track1

**Comment by JulianVolodia** [comment]
So, when shuffle mode activated, currently selected song should be not played first but change selection (shuffle) first and then play?
Is it that?

**Comment by staxas** [comment]
Yes, but I think what happened on my system was just a fluke. I can't
recreate the issue.

**Comment by staxas** [comment]
Sorry, the issue is when playing an album. When I double click on an album
in my library with 'shuffle all' mode on, the first song to be played is 1.
Consecutive songs are shuffled. This also happens for playing all songs in
a genre.

**Comment by JulianVolodia** [comment]
I will try to check this. Btw. this is reason why most issue tracker propose to attach reproduce steps. Even if non-deterministic (sometimes happens).

**Comment by JulianVolodia** [comment]
I confirm, but still don't have good idea (not enought time) how to resolve it. Sorry for long time to wait @staxas :((
